packages/cicero-core/package.json

```diff
@@ -89,6 +89,7 @@
     "json-stable-stringify": "1.0.1",
     "jszip": "3.2.1",
     "node-cache": "4.2.0",
+    "node-forge": "^0.10.0",
```

Comment:
Flagging new dependency to @dselman @jeromesimeon .

`node-forge` appears to be stable, well used and has a large number of contributors. However, there have been few contributions recently. The maintainers claim that the project is still active, https://github.com/digitalbazaar/forge/issues/897

Reply:
that's a good question. We may have inherited that from #496 . What are our options?

Reply:
Yes, you're right. We could find another library, although I'm not overly concerned. I just wanted to flag this so that we're all aware of the dependency.

Reply:
didn't even check until now, but the README indicates some browser support.